scio-bigquery/src/main/scala/com/spotify/scio/bigquery/client/JobOps.scala

```diff
@@ -94,7 +94,11 @@ private[client] final class JobOps(client: Client) {
         case (bqJob, jobReference) =>
           val jobId = jobReference.getJobId
           try {
-            val poll = client.underlying.jobs().get(client.project, jobId).execute()
+            val poll = client.underlying
+              .jobs()
+              .get(client.project, jobId)
+              .set("location", jobReference.getLocation)
```

Comment:
wouldn't this work as well?

```suggestion
              .setLocation(jobReference.getLocation)
```

Reply:
@seanhagen can you revise?

Reply:
Sure thing! ( Sorry, busy day yesterday )